File: 0026-remove-duplicates-from-sorted-array/0026-remove-duplicates-from-sorted-array.java
class Solution {
    public int removeDuplicates(int[] nums) {
        HashSet<Integer> hs=new HashSet<>();
        int x=0;
        for(int i=0;i<nums.length;i++){
            if(hs.contains(nums[i])){
                continue;
            }
            else{
                hs.add(nums[i]);
                nums[x]=nums[i];
                x++;
            }
        }
        return x;
    }
}

File: 0026-remove-duplicates-from-sorted-array/0026-remove-duplicates-from-sorted-array.py
class Solution:
    def removeDuplicates(self, nums: List[int]) -> int:
        #nums[:]=sorted(set(nums))
        a=[nums[0]]
        for i in range(len(nums)-1):
            if nums[i]<nums[i+1]:
                a.append(nums[i+1])
            if nums[i]==nums[i+1]:
                i+=1
        nums[:]=a
        
